vm/src/obj/objiter.rs

```diff
@@ -139,7 +136,7 @@ pub fn length_hint(vm: &VirtualMachine, iter: PyObjectRef) -> PyResult<Option<us
 #[pyclass]
 #[derive(Debug)]
 pub struct PySequenceIterator {
-    pub position: Cell<isize>,
+    pub position: AtomicCell<isize>,
```

Comment:
Why do you prefer the `AtomicCell` and not using `std::sync::atomic::AtomicIsize`?

Reply:
The first one I did was objstr, which had a non-promitive type `Option<i64>`, and I kept using it because it seemed more consistent; you know that it has the same functionality for the most part because it's the same base type with a different generic? I'm not sure, if you're really opposed to it I can change it, but as far as I can tell it's a zero-cost wrapper over the standard Atomic types, except with generics instead of completely separate types.

Reply:
Sounds good to me. Just wanted to understand the reason.